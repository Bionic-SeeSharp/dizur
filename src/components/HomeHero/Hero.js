import styled from 'styled-components';
import heroLg from './hero-lg.jpg';

const Hero = styled.div`
  background-image: url(${heroLg});
  height: 75vh;
  background-attachment: fixed;
  background-size: cover;
  text-align: center;
  flex-direction: column;
  display: flex;
  display: -webkit-flex;
  display: -ms-flexbox;
  align-items: center;
  -webkit-align-items: center;
  -ms-flex-align: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
`;

export default Hero;
