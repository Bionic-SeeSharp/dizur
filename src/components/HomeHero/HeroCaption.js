import styled from 'styled-components';

const HeroCaption = styled.h1`
  color: white;
  font-weight: bold;
  font-size: 44px;
  margin-top: 80px;
  padding-left: 2.5em;
  padding-right: 2.5em;
`;

export default HeroCaption;
