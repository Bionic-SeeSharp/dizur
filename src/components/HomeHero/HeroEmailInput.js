import styled from 'styled-components';

const HeroEmailInput = styled.input`
  width: 242px;
  height: 45px;
`;

export default HeroEmailInput;