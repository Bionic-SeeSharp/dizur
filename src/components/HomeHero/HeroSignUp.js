import React from 'react';
import { connect } from 'react-redux';
import { Control, Form } from 'react-redux-form';

const HeroSignUp = ({ submit }) => (
    {/*<Form model="signUpHero" onSubmit={submit}>
      <Control.text model=".email"/>
      <button>SIGN UP</button>
</Form>*/}
);

export default HeroSignUp;
