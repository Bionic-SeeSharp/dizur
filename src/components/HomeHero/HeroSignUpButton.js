import styled from 'styled-components';

const HeroSignUpButton = styled.button`
  border-radius: 25px;
  color: #00eb5f;
  height: 48px;
  width: 165px;
`;

export default HeroSignUpButton;
