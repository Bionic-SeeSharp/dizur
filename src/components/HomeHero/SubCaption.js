import styled from 'styled-components';

const SubCaption = styled.h4`
  color: white;
  font-weight: bold;
  font-size: 2em;
  word-spacing: -4px;
  margin-top: 20px;
  padding-left: 4em;
  padding-right: 4em;
`;

export default SubCaption;