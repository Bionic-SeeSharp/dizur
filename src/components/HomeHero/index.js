import React from 'react';
import Hero from './Hero';
import Wrapper from './Wrapper';
import HeroCaption from './HeroCaption';
import SubCaption from './SubCaption';
import LoginForm from '../LoginForm';

const HomeHero = () => (
  <Wrapper>
    <Hero>
      <HeroCaption>What music will you listen to today?</HeroCaption>
      <SubCaption>
        Check out your Deezer flow,<br />or browse curated music collections by
        genre.
      </SubCaption>
      <LoginForm />
    </Hero>
  </Wrapper>
);

export default HomeHero;
