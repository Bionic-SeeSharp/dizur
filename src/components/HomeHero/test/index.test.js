import HomeHero from '../index';
import React from 'react';
import { shallow } from 'enzyme';
import Wrapper from '../Wrapper';

describe('<HomeHero />', () => {
  it('render a HomeHero', () => {
    const wrapper = shallow(<HomeHero />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render a Wrapper component', () => {
    const renderedComponent = shallow(<HomeHero />);
    expect(renderedComponent.type()).toEqual(Wrapper);
  });
});
