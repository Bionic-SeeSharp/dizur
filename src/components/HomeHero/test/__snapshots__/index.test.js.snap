// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeHero /> render a HomeHero 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomeHero />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <styled.div>
        <styled.h1>
                What music will you listen to today?
        </styled.h1>
        <styled.h4>
                Be the first to know when we launch.
        </styled.h4>
</styled.div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <styled.h1>
            What music will you listen to today?
</styled.h1>,
          <styled.h4>
            Be the first to know when we launch.
</styled.h4>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "What music will you listen to today?",
          },
          "ref": null,
          "rendered": "What music will you listen to today?",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Be the first to know when we launch.",
          },
          "ref": null,
          "rendered": "Be the first to know when we launch.",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <styled.div>
          <styled.h1>
                    What music will you listen to today?
          </styled.h1>
          <styled.h4>
                    Be the first to know when we launch.
          </styled.h4>
</styled.div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <styled.h1>
              What music will you listen to today?
</styled.h1>,
            <styled.h4>
              Be the first to know when we launch.
</styled.h4>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "What music will you listen to today?",
            },
            "ref": null,
            "rendered": "What music will you listen to today?",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Be the first to know when we launch.",
            },
            "ref": null,
            "rendered": "Be the first to know when we launch.",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
